src/Presentation/UiBuilder.php

```diff
@@ -63,7 +59,7 @@ private function buildTabsViewModel( ?ItemId $selectedItemType ): array {
 
 		foreach ( $this->config->getItemTypes() as $itemType ) {
 			$tabs[] = [
-				'label' => $this->itemTypeLabelLookup->getLabel( $itemType ),
+				'label' => $this->facetLabelBuilder->getTabLabel( $itemType ),
```

Comment:
Same as the below comment. Old code was better, and `FacetLabelBuilder` should not have `getTabLabel`